tests/plugins/tool/docformatter_tool_plugin/valid_package/wrong.py

```diff
@@ -1,4 +1,4 @@
-'''
+"""
```

Comment:
Probably don't want to change formatting on this file. Was likely intentional to make sure our tools catch legitimate errors.